docker-compose.yml

```diff
@@ -49,6 +54,7 @@ compile:
   command: env REBAR=rebar3 make compile
   image: logplex_base
   volumes:
+    - ./_build:/usr/src/app/_build
```

Comment:
mounting your local _build directory could break lots of things if you have locally compiled dependencies. Logplex depends on some libraries that contain compiled NIFS that won't be cross-platform compatible.